server.js

```diff
@@ -279,19 +279,37 @@ router.post("/api/revalidate", bp.json(), async (req, res) => {
   // Find pull requests with matching account
   log.info("Received hook payload on revalidate endpoint");
   log.info(JSON.stringify(req.body, null, 2));
-  if (!req.body || !req.body.account || !req.body.account.nickname) {
+
+  if (!req.body || !req.body.event) {
     return error(res, "Invalid request sent to revalidate endpoint");
   }
-  if (!req.body.account.service === "github") {
-    return res.json({msg: "ignoring updates to accounts other than github"});
-  }
-  const ghUsername = req.body.account.nickname;
-  try {
-    const prs = await new Promise((res, rej) => store.getUnaffiliatedUserPRs(ghUsername, (err, prs) => {
+
+  if (req.body.event === "connected_account.created" && req.body.account && req.body.account.nickname) {  // new connected account
+    if (!req.body.account.service === "github") {
+      return res.json({msg: "Ignoring updates to accounts other than github"});
+    }
+    getPRs = new Promise((res, rej) => store.getUnaffiliatedUserPRs(req.body.account.nickname, (err, prs) => {
```

Comment:
getPRs should be declared beforehand (`let getPRs`);

Reply:
Fixed in https://github.com/w3c/ash-nazg/pull/184/commits/68351a0715a40f1fb88b06fe4537bf26738b36cd